pyvista/plotting/plotting.py

```diff
@@ -4652,7 +4662,18 @@ def remove_and_reactivate_prior_scalars(dsattr, name):
         for mesh, (name, assoc) in self._added_scalars:
             dsattr = mesh.point_data if assoc == 'point' else mesh.cell_data
             remove_and_reactivate_prior_scalars(dsattr, name)
+
+        # reactivate prior active scalars
+        for mesh, point_name, cell_name in self._prev_active_scalars:
+            if point_name is not None:
+                if mesh.point_data.active_scalars_name != point_name:
+                    mesh.point_data.active_scalars_name = point_name
```

Comment:
Probably minimal, but I can't be entirely sure as there might be some `Modified` which would lead to updates in the VTK pipeline. I'd rather leave it as is.